src/MSAL.PCL/Internal/Http/HttpClientFactory.cs

```diff
@@ -35,7 +35,17 @@ internal class HttpClientFactory
         private static HttpClient _client;
         private static readonly object LockObj = new object();
         public static bool ReturnHttpClientForMocks { set; get; }
+        public const long MaxResponseContentBufferSizeInBytes = 1024*1024;
 
+        private static HttpClient CreateHttpClient()
```

Comment:
You are right (again). [HttpClient has been designed to be re-used for multiple calls.](http://stackoverflow.com/questions/22560971/what-is-the-overhead-of-creating-a-new-httpclient-per-call-in-a-webapi-client)
Learn something new (to me) today.